js/src/chrome/projects/chrome.projects.js

```diff
@@ -624,6 +635,26 @@ Wu.Chrome.Projects = Wu.Chrome.extend({
 		var allUsers = _.sortBy(_.toArray(app.Users), function (u) {
 			return u.store.firstName;
 		});
+		
+		function onKeyUp(e) {
```

Comment:
@IgorZiegler  this function should be move to `this` namespace, see comment above.